fv3_cap.F90

```diff
@@ -15,11 +15,13 @@ module fv3gfs_cap_mod
 
   use ESMF
   use NUOPC
-  use NUOPC_Model,            only: model_routine_SS        => SetServices,       &
-                                    model_routine_Run       => routine_Run,       &
-                                    model_label_Advance     => label_Advance,     &
-                                    model_label_CheckImport => label_CheckImport, &
-                                    model_label_Finalize    => label_Finalize,    &
+  use NUOPC_Model,            only: model_routine_SS => SetServices,         &
+                                    routine_Run,                             &
+                                    label_Advertise,                         &
+                                    label_RealizeProvided,                   &
+                                    label_Advance,                           &
+                                    label_CheckImport,                       &
+                                    label_Finalize,                          &
```

Comment:
Add `label_TimestampExport`. See comment below.

Reply:
Added in 49d660f